### PR TITLE
Add math functions trunc, frac, rsqrt, lerp

### PR DESCRIPTION
**Pull request by andreaskoepf**
- [x] CPU implementation of trunc, frac, rsqrt, lerp
- [x] documentation
- [x] unit tests

### CODE REVIEW

File: lib/TH/THTensor.c [hunk 1]
```diff
@@ -7,6 +7,18 @@
 #include "THTensorDimApply.h"
 #include "THMath.h"
 
+inline double frac(double x) {
```
Comment (fmassa) [inline lib/TH/THTensor.c]: I wonder if those functions shouldn't go to the newly created [THMath.h](https://github.com/torch/torch7/blob/master/lib/TH/THMath.h). What do you think ?

**Comment by andreaskoepf** [comment]
@fmassa THMath.h is indeed a better place for the functions. thx

**Comment by andreaskoepf** [comment]
IMO it would be better if [genericSingleOpTest](https://github.com/torch/torch7/blob/master/test/test.lua#L44) was a 'real' function that would get the functions to test as arguments instead of using string-replacement...

**Comment by soumith** [comment]
they look good. squash and they're ready for commit.

**Comment by andreaskoepf** [comment]
rebased & squashed

**Comment by soumith** [comment]
Thanks Andreas!

**Pull request merged**